faker/providers/address/en_US/__init__.py

```diff
@@ -353,6 +373,24 @@ def zipcode_plus4(self):
         return "%s-%04d" % (self.zipcode(),
                             self.generator.random.randint(1, 9999))
 
+    def postcode_in_state(self, state_abbr='AL'):
```

Comment:
I would make `state_abbr` mandatory. Same for the other new methods.

Reply:
Sorry if this is dumb -- but it's not clear to me the best way to go about this. None of the provider functions have mandatory arguments. Removing the default value causes the doc generation to fail.

I was thinking of replacing the line with this, which seems to be in line with how the other provider functions are setup:

` 
def postcode_in_state(self, state_abbr=None):
    if state_abbr is None or state_abbr in self.states_abbr:
         state_abbr = state_abbr or self.random_element(self.states_abbr)
         ...
`

Does that make sense?